packages/protocol/src/action/client/ClientBdx.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2022-2025 Matter.js Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { RequestContext } from "#action/client/ClientInteraction.js";
+import { BdxMessenger } from "#bdx/BdxMessenger.js";
+import { Duration, WorkSlot } from "#general";
+
+export interface ClientBdxOptions {
```

Comment:
I'm curious why this is "Options" and not some type of "Request"

Reply:
I had it Request but bascially there are no real "request" parameter there beside specifying a timeout ... I can rename.

Reply:
renamed